lib/backend_common/backend_common/db.py

```diff
@@ -107,7 +107,7 @@ def init_database(app):
 
 
 def init_app(app):
-    app.db_prefix = app.name.replace('-', '_')
+    app.db_prefix = app.import_name
```

Comment:
i think we can skip this line completely and only use ``app.import_name``. looks like ``app.db_prefix`` is only used in this file.